src/Infrastructure/Zfs/ZfsServiceInterface.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Infrastructure\Zfs;
+
+interface ZfsServiceInterface
```

Comment:
What do you think about ?
```suggestion
interface FilesystemServiceInterface
```

Zfs is an implementation but FilesystemServiceInterface is maybe a too high level abstraction

Reply:
nice